transcription/transcription.go

```diff
@@ -76,3 +77,17 @@ func fileNameFromURL(url string) string {
 	fileName := tokens[len(tokens)-1]
 	return fileName
 }
+
+// Task returns a task function for transcription using transcription functions.
+func Task(audioURL string, emailAddresses []string) func() error {
```

Comment:
Added to issue https://github.com/hack4impact/audio-transcription-service/issues/27. Is there anything to test now or merge?